src/types.rs

```diff
@@ -480,8 +480,8 @@ impl Rewrite for ast::WherePredicate {
             ast::WherePredicate::RegionPredicate(ast::WhereRegionPredicate {
                 ref lifetime,
                 ref bounds,
-                ..
-            }) => rewrite_bounded_lifetime(lifetime, bounds, context, shape)?,
+                span,
+            }) => rewrite_bounded_lifetime(lifetime, bounds, span, context, shape).ok()?,
```

Comment:
Do we need to add a `rewrite_result` method for `ast::WherePredicate`? I see that this only updates the `rewrite` method.

Reply:
I added a `rewrite_result` for `ast::WherePredicate` in different branch. This pr does not update it.

Reply:
@ding-young Is there a PR open for those changes?

Reply:
I see you just opened https://github.com/rust-lang/rustfmt/pull/6309